### DISCUSSION TITLE
十分疑惑：项目无法打开问题？

### DISCUSSION THREAD

**Discussion by BoWang816** [umijs/umi#7039] [Q&A]
有时候项目启动后，一直在加载页面，浏览器直接卡死，控制台啥的都打不开；

清除了.umi, 重装了node_modules，重启了编译器，重启了电脑，kill了所有端口，但是还是无法启动，很玄学。。。

控制台也不报错，啥也不显示

不知道有人遇见过吗？

**Comment by dalang81** [comment]
我也是
node_modules要硬链接工程目录, 不让硬链接就死进程
现在是打开控制台, F5就死进程
